packages/template-retail-react-app/app/pages/account/profile.jsx

```diff
@@ -296,8 +286,8 @@ const PasswordCard = () => {
         <ToggleCard
             id="password"
             title={
-                <Skeleton height="30px" width="120px">
-                    {passwordHeadingText}
+                <Skeleton ref={headingRef} tabindex="-1" height="30px" width="120px">
```

Comment:
```suggestion
                <Skeleton ref={headingRef} tabIndex="-1" height="30px" width="120px">
```

Oops I missed this one. Other than this and commenting back in the check for server side rendering, PR looks good to me.

```
    // // Do not render anything in the account page on the server!
    // if (!onClient) {
    //     return null
    // }
```